src/ds5/ds5-auto-calibration.cpp

```diff
@@ -1159,6 +1241,12 @@ namespace librealsense
         {
             std::vector<uint8_t> res;
             rs2_metadata_type frame_counter = ((frame_interface*)f)->get_frame_metadata(RS2_FRAME_METADATA_FRAME_COUNTER);
+            rs2_metadata_type frame_ts = ((frame_interface*)f)->get_frame_metadata(RS2_FRAME_METADATA_FRAME_TIMESTAMP);
```

Comment:
Please consider changing this C style cast to C++ cast

Reply:
`rs_frame` object is a C-type, Part of the C-API in the hourglass pattern